tests/UI/campaigns/functional/BO/08_design/06_imageSettings/04_sortAndPagination.js

```diff
@@ -0,0 +1,233 @@
+    await helper.closeBrowserContext(browserContext);
+  });
+
+  it('should login in BO', async function () {
+    await loginCommon.loginBO(this, page);
+  });
+
+  it('should go to image settings page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToImageSettingsPage', baseContext);
+
+    await dashboardPage.goToSubMenu(
+      page,
+      dashboardPage.designParentLink,
+      dashboardPage.imageSettingsLink,
+    );
+
+    await imageSettingsPage.closeSfToolBar(page);
+
+    const pageTitle = await imageSettingsPage.getPageTitle(page);
+    await expect(pageTitle).to.contains(imageSettingsPage.pageTitle);
+  });
+
+  it('should reset all filters and get number of image types in BO', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'resetFilterFirst', baseContext);
+
+    numberOfImageTypes = await imageSettingsPage.resetAndGetNumberOfLines(page);
+    await expect(numberOfImageTypes).to.be.above(0);
+  });
+
+  // 1 : Create 15 new image settings
+  const creationTests = new Array(15).fill(0, 0, 15);
+
+  creationTests.forEach((test, index) => {
+    describe(`Create image setting n°${index + 1} in BO`, async () => {
+      const createImageTypeData = new ImageTypeFaker({name: `todelete${index}`});
+
+      it('should go to add new image type page', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'goToAddImageTypePage', baseContext);
+
+        await imageSettingsPage.goToNewImageTypePage(page);
+        const pageTitle = await addImageTypePage.getPageTitle(page);
+        await expect(pageTitle).to.contains(addImageTypePage.pageTitleCreate);
+      });
+
+      it('should create image type and check result', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'createImageType', baseContext);
+
+        const textResult = await addImageTypePage.createEditImageType(page, createImageTypeData);
+        await expect(textResult).to.contains(imageSettingsPage.successfulCreationMessage);
+
+        const numberOfImageTypesAfterCreation = await imageSettingsPage.getNumberOfElementInGrid(page);
+        await expect(numberOfImageTypesAfterCreation).to.be.equal(numberOfImageTypes + 1 + index);
+      });
+    });
+  });
+
+  // 2 : Pagination
+  describe('Pagination next and previous', async () => {
+    it('should change the item number to 20 per page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'changeItemNumberTo20', baseContext);
+
+      const paginationNumber = await imageSettingsPage.selectPaginationLimit(page, '20');
+      expect(paginationNumber).to.equal('1 2');
+    });
+
+    it('should click on next', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'clickOnNext', baseContext);
+
+      const paginationNumber = await imageSettingsPage.paginationNext(page);
+      expect(paginationNumber).to.equal('1 2');
```

Comment:
I think here you could check which page is active BY checking text in active item of the pagination list 
![image](https://user-images.githubusercontent.com/48441421/93235913-7c34da80-f77e-11ea-97e4-bb7adfc8734a.png)

Reply:
Done thx :+1: